packages/anoncreds/src/AnonCredsModule.ts

```diff
@@ -38,6 +38,20 @@ export class AnonCredsModule implements Module {
     dependencyManager.registerSingleton(AnonCredsLinkSecretRepository)
   }
 
+  public async initalize(agentContext: AgentContext) {
```

Comment:
```suggestion
  public async initialize(agentContext: AgentContext) {
```

Agent core checks for `initialize` method existence to call it during initialization process.

Reply:
Fixed